Makefile

```diff
@@ -177,7 +177,7 @@ gosec:
 
 .PHONY: gosec-scan
 gosec-scan: gosec
-	$(GOSEC) -fmt sonarqube -out gosec.json -no-fail -exclude-dir=.go ./...
+	$(GOSEC) -fmt sonarqube -out gosec.json -exclude-dir=.go -exclude-dir=test ./...
```

Comment:
I'm trying to decide whether removing `-no-fail` would affect `stolostron`, but it's probably nicer (and less complicated) to find this out in the action than from SonarCloud.

Reply:
I was thinking that in `stolostron`, we would not remove `-no-fail`, and we would just be alerted to issues by SonarCloud like we are now. But finding out earlier in the action might be good...

Doing it this way would let us possibly allow "low" severity issues in PRs here, but have SonarCloud track them separately. But I don't know if that's really useful.

Reply:
I feel like if SonarCloud is going to fail, we might as well have it fail here also?

Also I'm not sure excluding the test dir is necessary?
> gosec will ignore test files across all packages

Reply:
Some of our test utilities are not in `_test.go` files, so it doesn't know to ignore them. In particular, without this, it was noticing a "major" issue because of how we call `kubectl`